src/index.tsx

```diff
@@ -1077,7 +1077,7 @@ export class App extends React.Component<{}, AppState> {
 
             if (elementAtPosition && isTextElement(elementAtPosition)) {
               elements = elements.filter(
-                element => element !== elementAtPosition
+                element => element.id !== elementAtPosition.id
```

Comment:
Not necessary btw

Reply:
I just did it to not rely on reference comparisons in general. I can revert it back if you want.

Reply:
Just wanted to point it out because it technically borders on voodoo (nothing wrong with reference comparisons where it makes sense --- previous implem of `indexOf` does the same thing), but you can leave it :).

Reply:
Ref comparisons are also slightly faster (you don't need to access a prop and compare strings).

Reply:
When I think of immutability, I try to avoid reference comparisons at all costs, even if they are for read only accesses like above.

Reply:
Why? Reference comparisons are the bread and butter of immutability. That's how React bails on updates.